CMakeLists.txt

```diff
@@ -43,6 +43,11 @@ target_include_directories(cpptoml INTERFACE
   $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
   $<INSTALL_INTERFACE:include>)
```

Comment:
While at it, we might want to use GNUInstallDirs for this so that `CMAKE_INSTALL_INCLUDEDIR` is respected, see https://github.com/jtojnar/cmake-snips#hardcoding-the-installation-paths.